website/docs/r/ovh_domain_zone_record.html.markdown

```diff
@@ -46,9 +46,9 @@ The following attributes are exported:
 * `ttl` - The TTL of the record
 
 ## Import
+In order to import in terraform an existing zone record, you will need its _{id}_ which can be retrieved by using [OVH API portal](https://api.ovh.com/console/#/domain/zone/%7BzoneName%7D/record~GET)
 
-OVHcloud record can be imported using the `id` and the `zone`, eg:
 
 ```bash
-$ terraform import ovh_domain_zone_record.test 1234OVH_ID.zone.tld
+$ terraform import ovh_domain_zone_record.test {id}.{zone}
```

Comment:
```suggestion
$ terraform import ovh_domain_zone_record.test id.zone
```

Reply:
If we want to use `{}`format for input fields in import command, it should be useful to change it in every documentation pages :)